.github/workflows/build.yml

```diff
@@ -235,6 +235,13 @@ jobs:
           jupyter lite build --contents content --output-dir dist
 
       - name: Upload artifact
+        if: github.ref != 'refs/heads/main'
+        uses: actions/upload-artifact@v4
+        with:
+          path: ./lite/dist
+
+      - name: Upload github-pages artifact
+        if: github.ref == 'refs/heads/main'
         uses: actions/upload-pages-artifact@v1
         with:
           path: ./lite/dist
```

Comment:
Are those lite build artifacts not already available through the upload with `actions/upload-pages-artifact@v1`?

Reply:
yeah, I didn't realize that I could set the retention days for this action, the default value is just 1 day. Updating now